README.md

```diff
@@ -34,19 +34,21 @@ To update the installed JavaScript engines later on, just run `jsvu` again.
 
 ## Supported engines
 
-| JavaScript engine         | Binary name               | `mac64` | `win32`          | `win64`            | `linux32` | `linux64` |
-| ------------------------- | ------------------------- | ------- | ---------------- | ------------------ | --------- | --------- |
-| [**Chakra**][ch]          | `chakra` or `ch`          | ✅      | ✅               | ✅                 | ❌        | ✅        |
-| [**JavaScriptCore**][jsc] | `javascriptcore` or `jsc` | ✅      | ✅ <sup>\*</sup> | ✅ <sup>(32)\*</sup> | [❌][jsc] | [❌][jsc] |
-| [**SpiderMonkey**][sm]    | `spidermonkey` or `sm`    | ✅      | ✅               | ✅                 | ✅        | ✅        |
-| [**V8**][v8]              | `v8`                      | ✅      | ✅               | ✅                 | ✅        | ✅        |
+| JavaScript engine         | Binary name               | `mac64`             | `win32`         | `win64`              | `linux32` | `linux64` |
+| ------------------------- | ------------------------- | ------------------- | --------------- | -------------------- | --------- | --------- |
+| [**Chakra**][ch]          | `chakra` or `ch`          | ✅                  | ✅               | ✅                   | ❌        | ✅        |
+| [**JavaScriptCore**][jsc] | `javascriptcore` or `jsc` | ✅                  | ✅ <sup>\*</sup> | ✅ <sup>(32)\*</sup> | [❌][jsc] | [❌][jsc] |
+| [**SpiderMonkey**][sm]    | `spidermonkey` or `sm`    | ✅                  | ✅               | ✅                   | ✅        | ✅        |
+| [**V8**][v8]              | `v8`                      | ✅                  | ✅               | ✅                   | ✅        | ✅        |
+| [**XS**][xs]              | `xst`                     | ✅ <sup>(32)</sup>  | ✅               | ✅ <sup>(32)</sup>   | ✅        | ✅        |
```

Comment:
During installation, let’s name the binaries `xs` rather than `xst`. jsvu uses the names of the engines as binary names.

Reply:
I am not sure what you want me to do... xst is the name of the CLI.

Reply:
The V8 binary is originally called `d8`. jsvu considers this an implementation detail and exposes the binary as `v8` to avoid end-user confusion.

Reply:
I’ve amended the PR to show what I mean.